Metadata/AmazonMetadataBuilder.php

```diff
@@ -114,6 +115,18 @@ protected function getDefaultMetadata()
      */
     protected function getContentType($filename)
     {
+        // NEXT_MAJOR: Remove this BC hack
```

Comment:
stable ? no but this is unstable and on unstable, as long as you provide an upgrade note, it's fine by me.